cms/templates/product_page.html

```diff
@@ -11,7 +11,13 @@
     <meta property="product:brand" content="xPro">
     <meta property="product:availability" content="in stock">
     <meta property="product:condition" content="new">
-    <meta property="product:price:amount" content="{{ page.product.current_price }}">
+    {% if page.is_external_course_page and page.price %}
+      <meta property="product:price:amount" content="{{ page.price }}">
+    {% elif page.product.current_price %}
+      <meta property="product:price:amount" content="{{ page.product.current_price }}">
+    {% elif page.is_course_page and page.product.first_unexpired_run.current_price %}
```

Comment:
Check `page.product.first_unexpired_run` before trying to access its `current_price`.

Reply:
is it make any problem because same thing is happening in metadata-tiles.html ?